From: BunioFH <buniofh@gmail.com>
Date: Mon, 03 Jun 2026 13:07:01 +0200
Subject: Add missing gstreamer plugins for webaudio
Source: Comcast

Adds ffmpeg gstreamer plugin to browserlauncher environment construct

---
 BrowserLauncher/src/wpewebkit/wpewebkitconfig.cpp |    1 +
 1 file changed, 1 insertion(+)

Index: git/BrowserLauncher/src/wpewebkit/wpewebkitconfig.cpp
===================================================================
--- git.orig/BrowserLauncher/src/wpewebkit/wpewebkitconfig.cpp
+++ git/BrowserLauncher/src/wpewebkit/wpewebkitconfig.cpp
@@ -411,6 +411,7 @@ bool WpeWebKitConfig::setRialtoEnvironme
         "libgstaudiomixer.so",
         "libgstgio.so",
         "libgstinterleave.so",
+        "libgstlibav.so",
     };
 
     for (const std::string &pluginFileName : plugins)
